### PR TITLE
Added backend for authorization part

### PR DESCRIPTION
**Pull request by zaqbez39me**

### CODE REVIEW

File: fastapi_app/api/settings.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from datetime import timedelta
+
+from pydantic import (
+    BaseSettings
+)
+
+# Database settings
+DATABASE_URL = "postgresql+asyncpg://postgres:password@localhost:5432/postgres"
```
Comment (Dirakon) [inline fastapi_app/api/settings.py:8]: Better to extract it into a .env file somehow, to set it up with docker-compose later

File: README.md [hunk 2]
```diff
@@ -1,2 +1,23 @@
...
+   python -m venv /path/to/new/virtual/environment
+2. Activate venv depending on your OS.
+3. Install python requirements:
+   pip install -r requirements.txt
+4. Run server using unicorn:
+   fastapi_app.main:app --host=0.0.0.0 --port=8080 --reload
+
+
+### Link to the Postman collection with all the authorization requests and responses:
+![[Authorization collection](https://go.postman.co/workspace/Timers~1e9edfce-1e47-4521-8c6d-28315952eb9b/api/20118925-ecf8-4fe9-8584-b4dc59694c47)](https://img.shields.io/badge/Postman-FF6C37?style=for-the-badge&logo=postman&logoColor=white)
```
Comment (Dirakon) [inline README.md:21]: Can't click on it (goes to the page with the icon instead of going to the postman collection)

Comment (Dirakon) [inline README.md:21]:
```suggestion
[![Authorization collection](https://img.shields.io/badge/Postman-FF6C37?style=for-the-badge&logo=postman&logoColor=white)](https://go.postman.co/workspace/Timers~1e9edfce-1e47-4521-8c6d-28315952eb9b/api/20118925-ecf8-4fe9-8584-b4dc59694c47)
```

**Review by Dirakon — CHANGES_REQUESTED** [review]

File: fastapi_app/api/settings.py (hunk 1 above)
Comment (zaqbez39me) [reply fastapi_app/api/settings.py:8]: Will be done in future

**Review by Dirakon — APPROVED** [review]

**Pull request merged**